lib/libzrepl/data_conn.c

```diff
@@ -1221,22 +1263,41 @@ uzfs_zvol_rebuild_scanner(void *arg)
 				    "err(%d)", zinfo->name, rc);
 				goto exit;
 			}
-			bzero(&hdr, sizeof (hdr));
-			hdr.status = ZVOL_OP_STATUS_OK;
-			hdr.version = REPLICA_VERSION;
-			hdr.opcode = ZVOL_OPCODE_REBUILD_STEP_DONE;
-			zio_cmd = zio_cmd_alloc(&hdr, fd);
-			/* Take refcount for uzfs_zvol_worker to work on it */
-			uzfs_zinfo_take_refcnt(zinfo);
-			zio_cmd->zinfo = zinfo;
-			uzfs_zvol_worker(zio_cmd);
-			zio_cmd = NULL;
+
+			uzfs_zvol_send_zio_cmd(zinfo, &hdr,
+			    ZVOL_OPCODE_REBUILD_STEP_DONE,
+			    fd, NULL, 0, checkpointed_io_seq);
```

Comment:
lets send '0' instead of checkpointed_io_seq.. same at ALL_SNAP_DONE case also

Reply:
done